applications/serial_lte_modem/doc/TCPUDP_AT_commands.rst

```diff
@@ -191,6 +191,21 @@ Syntax
 * The ``<sec_tag>`` parameter is an integer.
   If it is given, a TLS client will be started.
   It indicates to the modem the credential of the security tag used for establishing a secure connection.
+* The ``<peer_verify>``parameter accepts the following values:
```

Comment:
```suggestion
* The ``<peer_verify>`` parameter accepts the following values:
```
Missing space, likely the cause.

Reply:
Thanks. I was completely blind for it. :)